cmd/cobra.go

```diff
@@ -0,0 +1,33 @@
+package caddycmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var rootCmd = &cobra.Command{
+	Use: "caddy",
+}
+
+const docsHeader = `Caddy is an extensible server platform.`
```

Comment:
This appears at the top of _every_ help command, e.g. `caddy help stop` -- I think this only needs to appear for `caddy help`, not subcommands

Reply:
Done!